config/default.yml

```diff
@@ -16,3 +16,6 @@ passwordResetToken:
 
 server:
   port: '8080'
+
+public:
+  currentLoginMethod: 'PHONE' #or 'EMAIL'
```

Comment:
Can we please keep current login method as `EMAIL`.

Also, rename this property to be `authenticationMechanism`